server/core/src/rodsServer.cpp

```diff
@@ -433,10 +567,10 @@ int main(int argc, char** argv)
         key_path_t{irods::CFG_ADVANCED_SETTINGS_KW, irods::CFG_HOSTNAME_CACHE_KW, irods::CFG_EVICTION_AGE_IN_SECONDS_KW},
         irods::get_hostname_cache_eviction_age());
 
-    /* start of irodsReServer has been moved to serverMain */
+    // Start of irodsReServer has been moved to serverMain.
     signal( SIGTTIN, SIG_IGN );
     signal( SIGTTOU, SIG_IGN );
-    signal( SIGCHLD, SIG_DFL ); /* SIG_IGN causes autoreap. wait get nothing */
+    signal( SIGCHLD, SIG_DFL ); // SIG_IGN causes autoreap. wait get nothing.
```

Comment:
Feel free to ignore this...

This comment seems like it's meant for the `SIG_IGN` line above? Also, what does "wait get nothing" mean?

Reply:
i didn't pay attention to that comment.

will consider adjusting or removing it.